interfaces/builtin/basedeclaration.go

```diff
@@ -355,6 +355,11 @@ slots:
       slot-snap-type:
         - core
     deny-auto-connection: true
+  realsense:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
```

Comment:
I'm probably missing something basic but how is this relating to the test that allows auto-connection between anything and anything?